spec/services/idv/phone_step_spec.rb

```diff
@@ -178,7 +189,7 @@
       it 'returns :warning' do
         subject.submit(phone: bad_phone)
         expect(subject.async_state.done?).to eq true
-        _result = subject.async_state_done(subject.async_state)
+        subject.async_state_done(subject.async_state)
```

Comment:
![chef_kiss](https://i.kym-cdn.com/photos/images/newsfeed/001/168/676/09e.gif)

Thanks for cleaning these up.